packages/block-library/src/archives/index.php

```diff
@@ -111,17 +120,19 @@ function register_block_core_archives() {
 		'core/archives',
 		array(
 			'attributes'      => array(
-				'showPostCounts'    => array(
-					'type'    => 'boolean',
-					'default' => false,
+				'align'             => array(
+					'type' => 'string',
+				),
+				'className'         => array(
```

Comment:
Not certain declaring this one is necessary here because there's a hook adding this attribute. Though one could think the hook is only declared on JavaScript which means nothing guarantees that it's available in PHP (which hopefully the server-side awareness work should resolve)